cpu/stm32l0/include/vendor/stm32l0xx.h

```diff
@@ -132,7 +132,15 @@
 /** @addtogroup Device_Included
   * @{
   */
-#if defined(STM32L011xx)
+#if defined(STM32L010xB)
+  #include "stm32l010xb.h"
+#elif defined(STM32L010x8)
+  #include "stm32l010x8.h"
+#elif defined(STM32L010x6)
```

Comment:
Those files are missing.

Reply:
I have just prepare those include to help people to fine the right one in the HAL.
I don't have carrier board for thoses CPU, so i can't test this part.

Do you prefers i remove this part ?

Reply:
Adapting the vendor files to what RIOT expects is only a small patch

```patch
--- stm32l010xx.h       2019-12-13 09:46:42.085251331 +0100
+++ stm32l010xx.h       2019-12-13 09:45:43.034789432 +0100
@@ -72,7 +72,7 @@
 /******  Cortex-M0 Processor Exceptions Numbers ******************************************************/
   NonMaskableInt_IRQn         = -14,    /*!< 2 Non Maskable Interrupt                                */
   HardFault_IRQn              = -13,    /*!< 3 Cortex-M0+ Hard Fault Interrupt                       */
-  SVC_IRQn                    = -5,     /*!< 11 Cortex-M0+ SV Call Interrupt                         */
+  SVCall_IRQn                 = -5,     /*!< 11 Cortex-M0+ SV Call Interrupt                         */
   PendSV_IRQn                 = -2,     /*!< 14 Cortex-M0+ Pend SV Interrupt                         */
   SysTick_IRQn                = -1,     /*!< 15 Cortex-M0+ System Tick Interrupt                     */
 
@@ -102,7 +102,6 @@
   */
 
 #include "core_cm0plus.h"
-#include "system_stm32l0xx.h"
 #include <stdint.h>
 
 /** @addtogroup Peripheral_registers_structures
@@ -414,10 +413,7 @@
   __IO uint32_t PSC;       /*!< TIM prescaler register,                       Address offset: 0x28 */
   __IO uint32_t ARR;       /*!< TIM auto-reload register,                     Address offset: 0x2C */
   uint32_t      RESERVED12;/*!< Reserved                                      Address offset: 0x30 */
-  __IO uint32_t CCR1;      /*!< TIM capture/compare register 1,               Address offset: 0x34 */
-  __IO uint32_t CCR2;      /*!< TIM capture/compare register 2,               Address offset: 0x38 */
-  __IO uint32_t CCR3;      /*!< TIM capture/compare register 3,               Address offset: 0x3C */
-  __IO uint32_t CCR4;      /*!< TIM capture/compare register 4,               Address offset: 0x40 */
+  __IO uint32_t CCR[4];    /*!< TIM capture/compare register 1,               Address offset: 0x34 */
   uint32_t      RESERVED17;/*!< Reserved,                                     Address offset: 0x44 */
   __IO uint32_t DCR;       /*!< TIM DMA control register,                     Address offset: 0x48 */
   __IO uint32_t DMAR;      /*!< TIM DMA address for full transfer register,   Address offset: 0x4C */
```

I've crerated a PR to this branch https://github.com/Power-Lan/RIOT/pull/1